tests/ci/common_fuzz.sh

```diff
@@ -105,7 +109,8 @@ function run_fuzz_test {
   # Step 2 merge any new files from the run corpus and GitHub src corpus into the shared corpus (EFS)
   time "./${FUZZ_TEST_PATH}" -merge=1 "$SHARED_CORPUS" "$FUZZ_TEST_CORPUS" "$SRC_CORPUS"
```

Comment:
At this point isn't SRC_CORPUS a subset of FUZZ_TEST_CORPUS?

Reply:
Yes, but merging in from GitHub is a nice way to bootstrap new fuzz tests. It also lets us pick up new corpus files from BoringSSL during merges.

Reply:
So step 1 didn't merge $SRC_CORPUS into $LOCAL_RUN_CORPUS?